docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -100,11 +101,15 @@ def downstream_asset(upstream_asset):
 
 defs = Definitions(
     assets=[upstream_asset, downstream_asset],
-    resources={"io_manager": s3_pickle_io_manager, "s3": s3_resource},
+    resources={
+        "io_manager": ConfigurablePickledObjectS3IOManager(
+            s3_resource=S3Resource(), s3_bucket="my-bucket"
+        ),
+    },
 )
 ```
 
-This example also includes `"s3": s3_resource` because the <PyObject module="dagster_aws.s3" object="s3_pickle_io_manager" /> depends on an S3 resource.
+This example also constructs a <PyObject module="dagster_aws.s3" object="S3Resource" /> because the <PyObject module="dagster_aws.s3" object="ConfigurablePickledObjectS3IOManager" /> depends on the S3 resource.
```

Comment:
hmm was thinking maybe worth linking to resource-resource dependency doc section for more details (https://docs.dagster.io/master/concepts/resources#resources-which-depend-on-other-resources) but then was debating myself because we no longer say "io managers are resources" so linking might be actually more confusing 🤔

Reply:
Agree on this, which is why IMO we should stick with the "IOManagers are resources" language.

Reply:
Added a reference to the resource-resource dep docs